GitExternal.cmake

```diff
@@ -14,13 +14,12 @@
 #    or in the given absolute path using the given repository and tag
 #    (commit-ish).
 #
+# Targets:
+#  * DIR-rebase: fetches latest updates and rebases the given external
```

Comment:
rename DIR-rebase to <directory>-rebase for consistency with the rest of the documentation